internal/pkg/skuba/kubernetes/versions.go

```diff
@@ -89,7 +89,7 @@ var (
 				ToolingVersion:          "0.1.0",
 			},
 			AddonsVersion: AddonsVersion{
-				Cilium:  &AddonVersion{"1.5.3", 0},
+				Cilium:  &AddonVersion{"1.5.3", 1},
```

Comment:
Can this be bumped on all Kubernetes versions? I know it's a little repetitive, but here we are favouring expliciteness.

Reply:
done!